src/js/components/DateInput/DateInput.js

```diff
@@ -74,8 +74,12 @@ const DateInput = forwardRef(
     useEffect(() => {
       if (
         schema &&
-        value &&
-        ((Array.isArray(value) && value[0]) || !Array.isArray(value))
+        (value || value?.length === 0) &&
```

Comment:
Could this be simplified to `value !== undefined`?

Reply:
Thank you for the review.
I have simplified that expression to the following:
 ```
![undefined, null].includes(value) &&
            Array.isArray(value) || !Array.isArray(value)
```
What do you think @ericsoderberghp ?